cmd/gendocs/gen_kube_docs.go

```diff
@@ -73,6 +76,10 @@ func main() {
 		// generate docs for kubelet
 		kubelet := kubeletapp.NewKubeletCommand()
 		doc.GenMarkdownTree(kubelet, outDir)
+	case "kubectl":
```

Comment:
I don't know if this is what we expected when we put kubectl into separate repo.

@shiywang